packages/idealo.design/package.json

```diff
@@ -30,6 +30,7 @@
     "cors": "^2.8.5",
```

Comment:
When I tried compiling & starting the app, I needed to manually install the npm packages `html-to-react` and `html-to-text` which were missing in package.json. What went wrong, since the default behavior of npm is to automatically add installed packages to package.json?

Reply:
We didn't add those packages to the package.json. We'll add them soon.